commands.js

```diff
@@ -745,6 +745,12 @@ exports.commands = {
 				return this.errorReply("/" + cmd + " - Access denied for promoting/demoting to " + Config.groups[nextGroup].name + ".");
 			}
 		}
+		if ((!room.isPrivate || !room.battle || !room.isPersonal) && targetUser.locked) {
+			Monitor.log("[CrisisMonitor] " + user.name + " was automatically demoted in " + room.id + " for trying to promote locked user: " + targetUser.name + ".");
+			delete room.auth[user.userid];
```

Comment:
sorry I did my updated commit around the time you made the "panpawn is right" comment so I didn't know if you wanted that or not